extensions/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts

```diff
@@ -0,0 +1,1654 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as request from 'request';
+import * as http from 'http';
+
+let defaultBasePath = 'https://localhost';
+
+let primitives = [
+	'string',
```

Comment:
Do you really need all this stuff? That's a lot of extra code that I'm pretty sure you don't need.

Reply:
This is generated code by Swagger. The RestfulAPI will be change soon which requires to generate new code. This will be replaced soon anyway.